app/models/author.rb

```diff
@@ -0,0 +1,10 @@
+class Author < ActiveRecord::Base
+  has_many :books
+
+  validates :name, presence: true
+  validates :biography, presence: true
+
+  def five_books
```

Comment:
Even it duplicate, but I feel it's just `Accidental duplication`[0] because the logic to pull number of books to show under author may different from logics to pull number of books to show under a list.

Another thing, I usually not having a method name which includes something that is variable which in this case is `5` because we may want to show 3 or 6 later, and we don't want to change the method name and all callers of this method when we just want to change the number. The problem we miss the abstraction here, we may just call it `top_books` or `popular_books`, and in the real app which will involve some logics to pull and base on ratings or reviews.

[0] http://www.justinweiss.com/articles/i-dry-ed-up-my-code-and-now-its-hard-to-work-with-what-happened/